process/peer/PeerTypeInfo.go

```diff
@@ -0,0 +1,22 @@
+package peer
+
+type PeerTypeInfo struct {
```

Comment:
Do we need to export the struct as we have getters already 
if yes, then add comment for exported struct

Reply:
refactored and added the PeerTypeInfo to the data package, without the PeerTypeInfoHandler